.github/workflows/browser-tests.yml

```diff
@@ -275,7 +265,6 @@ jobs:
                  {\"blocks\": [{\"type\": \"section\",\"text\": {\"type\": \"mrkdwn\",\"text\": \"
                  $RESULT_EMOJI *$GITHUB_REPOSITORY*:*$GITHUB_REF_NAME* ($GITHUB_ACTOR) |
                  <$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID|Details> |
```

Comment:
Our current notifications look like this:
```
ibexa/product-catalog:main (Steveb-p) | Details | Report (1/1)
```

If I understand correctly the notification will look like this:
```
ibexa/product-catalog:main (Steveb-p) | Details | (1/1)
```

Just a suggestion: what about something like this?
```
ibexa/product-catalog:main (Steveb-p) | Details | 1/1
```

Reply:
Last one should be ok 👍

Reply:
Done in https://github.com/ibexa/gh-workflows/pull/42/commits/3e225bb35788f4e5062988e98ff67a36aac9ebb6.